src/components/units-list/units-list-unit/units-list-unit.module.css

```diff
@@ -1,45 +1,51 @@
 .unit {
-  padding: 0 0 theme("spacing.2") theme("spacing.2");
+  padding: 6px 0;
+  padding-left: 14px;
+
+  &__selected {
+    border: 1px solid theme('colors.grey.dark');
+    border-radius: 4px;
+  }
 }
 
 .unit-title {
   display: flex;
   align-items: center;
 
   &__title {
+    @apply typography-s;
+
     margin-right: 4px;
-    color: theme("colors.black");
+    color: theme('colors.black');
```

Comment:
Single quotes should have caused an Stylelint error. Check your precommit hooks please

Reply:
we have single quotes set for strings